test/e2e/catalog_e2e_test.go

```diff
@@ -1309,6 +1309,179 @@ var _ = Describe("Starting CatalogSource e2e tests", func() {
 			}
 		}
 	})
+
+	When("A CatalogSource is created with a properly formed CSV", func() {
+		var (
+			og               operatorsv1.OperatorGroup
+			namespaceName    string
+			packageName      string
+			stableChannel    string
+			subscriptionName string
+			manifests        []registry.PackageManifest
+			crd              apiextensions.CustomResourceDefinition
+			csv              v1alpha1.ClusterServiceVersion
+			cs               *v1alpha1.CatalogSource
+		)
+
+		BeforeEach(func() {
+			namespaceName = genName("good-csv-")
+			og = operatorsv1.OperatorGroup{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      fmt.Sprintf("%s-operatorgroup", namespaceName),
+					Namespace: namespaceName,
+				},
+			}
+			ns = SetupGeneratedTestNamespaceWithOperatorGroup(namespaceName, og)
```

Comment:
Can we piggy back off the top-level namespace that's been already created for us before this test case runs: https://github.com/operator-framework/operator-lifecycle-manager/blob/4be8b0c5ad2fd8cebe041ce9d7992019b8f02fbb/test/e2e/catalog_e2e_test.go#L54-L61?

Reply:
Turns out this is totally doable. Will delete redundant work.